packages/react-component-library/package.json

```diff
@@ -17,6 +17,8 @@
   "license": "MIT",
   "scripts": {
     "build": "concurrently \"webpack -p --config=webpack/prod.js\" \"yarn types\"",
+    "build:dev": "concurrently \"webpack -p --config=webpack/dev.js\" \"yarn types\"",
+    "build:analyze": "concurrently \"webpack -p --config=webpack/analyze.js\" \"yarn types\"",
     "dev": "concurrently \"webpack --watch --config=webpack/dev.js\" \"yarn types --watch\"",
```

Comment:
Has this `dev` task been superseded by `build:dev`?

Reply:
no. Dev watches for changes, the build dev builds a bundle that is not super compressed and means sensible error messages when tests fail in docs